additive-number.py

```diff
@@ -0,0 +1,33 @@
+class Solution:
+    def isAdditiveNumber(self, num: str) -> bool:
+        s = num
```

Comment:
S is not a very descriptive name, it best be replaced with something more descriptive